### ISSUE TITLE
Improve Bad Map Alert

### ISSUE THREAD

**Issue by MuffinSpawn** [MuffinSpawn/FMS#166]
Alerts indicating a failed map were not noticed during testing at Argonne.

Make the error an error.dat in addition to being included in the map.nack.
Make sure that the verification is still working.
Log details of bad maps so that we can track down the problem. This should include the reflector IDs, LTCS coordinates, prediction residuals (inversely mapped DSCS vs measured LTCS). The DSCS coordinates can be retrieved from the configuration INI.

**Comment by MuffinSpawn** [comment]
I have modified Position to also pass through any mapping error so that the MAP case triggers an error.dat.
I added a logging VI that outputs the reference reflector IDs, shoot statuses, LTCS and DSCS coordinate lists, and transformed/measured cartesian LTCS residuals. All coordinate lists are in Python format for ease of importing into an analysis script.

I still need to test this with a reference network that has at least 5 reflectors.
